recipes/baderkit/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "baderkit" %}
+{% set version = "0.1.1" %}
```

Comment:
You need to add a `{% set python_min = "3.10" %}` and then use the `python_min` jinja variable below as shown in:
https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python